script/ensure-node-source-fixture-files.js

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const https = require('https');
+
+// This "nodeVersion" specifies the version of Node used by certain specs.
+// Update nodeVersion here to run those specs against newer Node source files.
+// (May be needed to work around build issues with newer Node bundled with ppm.)
+// This string is also hard-coded in those spec files, so update it there, too.
+const nodeVersion = 'v18.12.1'
```

Comment:
```suggestion
const nodeVersion = "v18.12.1";
```
Codacy suggestion: Single quote -> Double quote
Codacy suggestion: End semicolon